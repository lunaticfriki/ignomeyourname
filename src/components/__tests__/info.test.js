import React from 'react'
import renderer from 'react-test-renderer'
import Info from '../info'

describe('Info component suite', () => {
  const dummyProps = {
    gnomes: [],
    gnome: {},
  }
  test('Info snapshot', () => {
    const component = renderer.create(<Info {...dummyProps} />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
