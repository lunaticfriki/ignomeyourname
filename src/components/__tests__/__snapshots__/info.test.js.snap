// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component suite Info snapshot 1`] = `
<div
  className="sc-AxjAm jnvo"
/>
`;
